src/matchcode_toolkit/fingerprinting.py

```diff
@@ -226,7 +226,7 @@ def get_file_fingerprint_hashes(location, ngram_length=8, window_length=64, **kw
 
 def create_file_fingerprints(content, ngram_length=8, window_length=64):
```

Comment:
@pombredanne What should I change these defaults to? In the tests, I am using a window_length of 5.